datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -0,0 +1,80 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+from os import path
+
+import click
+
+from ...utils import complete_valid_checks, get_root, get_valid_integrations, read_readme_file
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+IMAGE_EXTENSIONS = {".png", ".jpg"}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate README.md files')
+@click.pass_context
+@click.argument('integration', autocompletion=complete_valid_checks, required=False)
+def readmes(ctx, integration):
+    """Validates README files
+
+    If `check` is specified, only the check will be validated,
+    otherwise all README files in the repo will be.
+    """
+
+    repo = ctx.obj['repo_name']
+
+    errors = False
+    integrations = []
+
+    if integration:
+        integrations = [integration]
+    else:
+        integrations = sorted(get_valid_integrations())
+
+    for integration in integrations:
+        has_overview = False
+        has_setup = False
+
+        lines = read_readme_file(integration)
+        for line_no, line in lines:
+
+            if "## Overview" == line.strip():
+                has_overview = True
+
+            if "## Setup" == line.strip():
+                has_setup = True
+
+            for ext in IMAGE_EXTENSIONS:
+                if ext in line:
+                    IMAGE_REGEX = (
+                        rf".*https:\/\/raw\.githubusercontent\.com\/DataDog\/"
+                        rf"{re.escape(repo)}\/master\/({re.escape(integration)}\/images\/.*.{ext}).*"
+                    )
+
+                    match = re.match(IMAGE_REGEX, line)
+                    if not match:
+                        errors = True
+                        echo_failure(f"{integration} readme file does not have a valid image file on line {line_no}")
```

Comment:
One thing I noticed is that the other validations print the integration name once when there is an error, then print each error message on a newline with a slight indent, to group errors per integration) 
Example in the manifest validations - https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py#L289-L301

Reply:
I like that format! I updated the PR